src/sleep/sleep.c

```diff
@@ -23,11 +24,13 @@
 #include "log.h"
 #include "main-func.h"
 #include "parse-util.h"
+#include "poll.h"
```

Comment:
system headers should be included with `<>`, i.e.

```c
#include <poll.h>
```

(and then moved to the top)